.babelrc

```diff
@@ -1,4 +1,19 @@
 {
-  "presets": ["env", "react", "stage-1"],
-  "plugins": ["add-module-exports"]
+  "env": {
+    "development": {
```

Comment:
Can we remove this code duplication by using `es5` and `es6` as `BABEL_ENV` explicitly?

Reply:
good call out! I missed that on my initial review